force-app/main/default/permissionsets/PMDM_View.permissionset-meta.xml

```diff
@@ -4,6 +4,18 @@
         <application>Program_Management</application>
         <visible>true</visible>
     </applicationVisibilities>
+    <classAccesses>
+        <apexClass>AttendanceController</apexClass>
+        <enabled>true</enabled>
+    </classAccesses>
+    <classAccesses>
+        <apexClass>FieldSetController</apexClass>
+        <enabled>true</enabled>
+    </classAccesses>
+    <classAccesses>
+        <apexClass>RecentServiceSessionController</apexClass>
+        <enabled>true</enabled>
+    </classAccesses>
```

Comment:
these are already on the other perm sets?

Reply:
Indeed!